Exercise_A/index.html

```diff
@@ -20,19 +27,135 @@
   </head>
 
   <!-- Nav bar setup -->
+ 
 
   <body data-spy="scroll" data-target=".navbar">
     <!-- data-spy="scroll" data-target=".navbar" data-offset="50" -->
+
     <!--Allows us to track the progress of the scroll -->
+
     <!-- Nav bar items -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light sticky-top">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#index.html">The Only bottstrap page</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarText">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="#About">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#Fouriconshead">Four Icons</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#Bullshit">Bullshit</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav">
+        <li>
+          <a class="page scoll" herf="#about">About</a>
+        </li>
+        <li>
+          <a class="page scoll" herf="#Fouriconshead">Four Icons</a>
+        </li>
+        <li>
+          <a class="page scoll" herf="#Bullshit">Bullshit</a>
+        </li>
+      </ul>
+    </div>
+    </nav>
 
     <!-- Can use Jumbotron -->
+    <div>
+      <div class="header-content header">
+          <div class="header-content-inner">
+              <h1>Hey Look, It's Every Bootstrap Website Ever</h1>
+              <hr>
+              <p>Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!</p>
+              <a href="#about" class="bodybutton">This button makes the page scroll</a>
+          </div>
+      </div>
+    </div>
     <!-- Second Section -->
-
+<Div class="about bg-warning header-content">
+  <div class="header-content-inner">
+  <h1>Want to make an original website yourself?</h1>
+  <hr>
+  <p>Forget that! Who would ever want to put in all of that effort for a website? Just open up your web browser and type "bootstrap template" into your favorite search engine, like Yahoo! or Bing, and you're on your way! There are hundreds of templates to choose from, but go ahead and pick this same exact template from the first result on google, edit a few lines, and you're on your way! No one will notice!</p>
+  <a class="bodybutton">Google that</a>
+  </div>
+</Div>
     <!-- Four icons -->
+<Div class="container Fouriconshead">

```

Comment:
Decent implementation here.

Do not use capital letters within your HTML element tags
it is 
<div> 
NOT
<Div>

Though it works this is unclean code